doc/src/manual/arrays.md

```diff
@@ -237,8 +237,73 @@ indices.
 
 If all indices are vectors, for example, then the shape of `X` would be `(length(I_1), length(I_2), ..., length(I_n))`,
 with location `(i_1, i_2, ..., i_n)` of `X` containing the value `A[I_1[i_1], I_2[i_2], ..., I_n[i_n]]`.
+
+Example:
```

Comment:
Out of curiosity, why illustrate indexing behavior with a four-dimensional array rather than something simpler?

Reply:
I wanted to show the squeezing of various dimensions and what mixing index types does (kind of hard to do with just a 2D array) and I like 2^4.

Reply:
Fair enough :). `2^4` is special.

Reply:
do you want to `Approve` :D?